liquidcore/home/discover.py

```
@@ -0,0 +1,20 @@
+import requests
+from django.conf import settings
+from requests.exceptions import ConnectionError
+from json.decoder import JSONDecodeError
+
+def get_status():
+    if not settings.DISCOVERY_URL:
+        return {"running": False, "error": "discovery_url_not_configured"}
```

Alex is saying we should throw a sad exception on startup.